db/migrate/20210709062259_create_comments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[6.1]
+  def change
+    create_table :comments do |t|
+      t.string :body
+      t.integer :product_id
```

Comment:
Why we have only 'product_id' column with type integer?
This is a relation?
We need to use 'reference' if we need to create a relation between tables